pkg/strings/unescape_internal_test.go

```diff
@@ -0,0 +1,84 @@
+package strings_test
+
+import (
+	"testing"
+
+	"github.com/plgd-dev/hub/v2/pkg/strings"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUnescape(t *testing.T) {
+	tbl := []struct {
```

Comment:
@coderabbitai Add inputs here to test `UnescapingModeAllExceptReserved` and `UnescapingModeAllExceptSlash` modes

Reply:
@coderabbitai Yes, write those inputs that you mentioned.

Reply:
@coderabbitai Make commitable suggestions from the above code

Reply:
@coderabbitai Is there a way I can allow you to make commitable suggestions that I can approve without having to manually add the code?